.github/workflows/run_alembic_migrations.yml

```diff
@@ -0,0 +1,64 @@
+name: Automated Migrations
+
+on:
+  push:
+    branches: [ main ]
+jobs:
+  run-alembic-migrations-ssh:
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+        SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
+        SSH_HOST: ${{secrets.SSH_HOST}}
+        SSH_PORT: ${{secrets.SSH_PORT}}
+        SSH_USER_NAME: ${{secrets.SSH_USER_NAME}}
+        DATABASE_URL: ${{secrets.MIGRATION_DATABASE_URL}}
+        MIGRATION_API_ENDPOINT: ${{secrets.MIGRATION_API_ENDPOINT}}
+        MIGRATION_API_TOKEN: ${{secrets.MIGRATION_API_TOKEN}}
+        # Check if the migration API params are present we will use the migration API to run migrations.
+        USE_MIGRATION_API: ${{ (secrets.MIGRATION_API_ENDPOINT != '') && (secrets.MIGRATION_API_TOKEN != '') }}
+        # Check if the database URL is present we will use the database URL to run migrations.
```

Comment:
`If database URL is present, run migrations on DB connection`

Reply:
Done